### ISSUE TITLE
When Breath Becomes Air

### ISSUE THREAD

**Issue by phillipou94** [phillipou94/react-blog#3]
![When Breath Becomes Air](https://cdn.substack.com/image/fetch/w_1100,c_limit,f_auto,q_auto:good,fl_progressive:steep/https://bucketeer-e05bbc84-baa3-437e-9518-adb32be77984.s3.amazonaws.com/public/images/39573303-7b23-4187-8fd8-be95ded2a0a6_3024x3024.jpeg)

A few years ago, a friend from work introduced me to Paul Kalanithi’s memoir, _[When Breath Becomes Air](https://www.amazon.com/When-Breath-Becomes-Kalanithi-Paul/dp/1784701998/ref=asc_df_1784701998/?tag=hyprod-20&linkCode=df0&hvadid=241920322233&hvpos=&hvnetw=g&hvrand=14496999507785134956&hvpone=&hvptwo=&hvqmt=&hvdev=c&hvdvcmdl=&hvlocint=&hvlocphy=1014221&hvtargid=pla-406163982473&psc=1)._ What was supposed to be just a casual catchup turned into my friend passionately explaining how deeply this book impacted him. Between bites of food, he walked me through his new perspectives on health, his career, and what makes life worthwhile.

He even claimed that this book still makes him cry despite having read it multiple times.

I nodded nonchalantly and, hoping to bring some levity to some pretty heavy topics, asked whether he wanted to grab drinks after work.

He declined.

Apparently, this book had made him more mindful of his health and he no longer intends to drink on weekdays. (“But going out on weekends is still okay though” he interjected quickly after).

I accepted what was one of the strangest social rejections I've ever received and made a mental note to pickup this book after work.

_I’ve never cried from reading a book, how emotional can this thing be?_

**Spoiler:** I did cry. I cried a lot more than I’m comfortable admitting.

# Background

I’ve never read a book where every sentence feels so precise and intentional.

Paul’s personal story battling cancer requires impeccably skillful writing to authentically communicate his feelings of pain, hope, and purpose.

Before becoming a neurosurgeon, Paul Kalanithi was driven to answer what actually makes people’s lives “meaningful”. At Stanford, he pursued degrees in both English literature and human biology with the idea that both of those fields would provide different but complementary perspectives on the human experience.

Paul showcases this unique educational background by thoughtfully exploring the intersections of language, biology, and the human condition.

I finished his book with a deeper appreciation for doctors, a greater understanding of how fragile our lives are, and motivation to reflect on what makes _my_ life personally meaningful.

This is my most cited "favorite book" so, given the spoilers ahead, I'd encourage you to read it for yourself!

----------
# TL;DR

> [L]ife wasn’t about avoiding suffering.
> -Paul Kalanithi

# Favorite Takeaways

### 1: The difference between **a job vs. a calling**

> People often ask if [being a neurosurgeon] is a calling, and my answer is always yes. You can't see it as a job, because if it's a job, it's one of the worst jobs there is.
> 
> -   Paul Kalanithi

Up until I was about 20 years old, my mother **strongly** encouraged me to become a doctor. The nudges encouragement ranged from subtle nudges like taking me to the biology section of the public library to more explicit career advice: “Being an NBA player is great, but do you know what profession also makes a lot of money?!”

With the exception of one summer where I watched House MD religiously, I was never shy about expressing my disinterest towards medicine and that resistance only grew the more my mother encouraged me to warm up to that career path.

It resembled a fairly cliche Chinese mother/American-born son dynamic but, with the benefit of hindsight, I could see why that future for me was alluring for her.

**Funny Side Story:** I have so much appreciation for doctors and nurses! I had to get surgery in 2019 and I was so high off Vicodin I told my surgical team that I loved them 😂

![https://cdn.substack.com/image/fetch/w_1100,c_limit,f_auto,q_auto:good,fl_progressive:steep/https%3A%2F%2Fbucketeer-e05bbc84-baa3-437e-9518-adb32be77984.s3.amazonaws.com%2Fpublic%2Fimages%2F25203029-2c01-4706-b61c-6e069ba2cebb_3264x2448.jpeg](https://cdn.substack.com/image/fetch/w_1100,c_limit,f_auto,q_auto:good,fl_progressive:steep/https%3A%2F%2Fbucketeer-e05bbc84-baa3-437e-9518-adb32be77984.s3.amazonaws.com%2Fpublic%2Fimages%2F25203029-2c01-4706-b61c-6e069ba2cebb_3264x2448.jpeg)

Doctors get paid well and command the social cache of practicing one of the noblest professions in our society. If you asked my immigrant parents what they would want for their only son, I’m sure financial stability and becoming a respected member of your community would be high on that wishlist.

> Indeed, this is how 99 percent of people select their jobs: pay, work environment, hours. But that’s the point. Putting lifestyle first is how you find a job—not a calling.
> 
> -Paul Kalanithi

It’s easy to fixate on the external validators of being in medicine, but it obscures the astounding sense of duty and responsibility doctors and nurses bear everyday.

I don’t think any amount of money or status make the long hours seem shorter, the life or death decisions simpler, or the difficult conversations easier. Paul's dedication to his work made me realize how truly great doctors are motivated by profound passion and purpose.

> Good intentions were not enough, not when so much depended on my skills, when the difference between tragedy and triumph was defined by one or two millimeters.
> 
> -Paul Kalanithi

Imagine going to work every day knowing that an “off day” means someone losing their life. Perhaps worse, imagine making a mistake where the patient is still technically "alive" but they lose all cognitive ability to enjoy their life and relationships.

> Because the brain mediates our experience of the world, any neurosurgical problem forces a patient and family, ideally with a doctor as a guide, to answer this question: What makes life meaningful enough to go on living?
> 
> -Paul Kalanithi

Would you stop having a functional right hand in order to stop having seizures? Would you exchange your mother’s ability to understand language in order to keep her from a coma? How much neurological suffering would you let your child endure before admitting that death is preferable?

Paul faces these difficult questions often. For him, being a neurosurgeon isn't just a job trying to "fix" patients.

He bears the much greater responsibility of supporting patients physically, emotionally, and spiritually through some of their most difficult periods of their lives.

> [M]y highest ideal was not saving lives — everyone dies eventually — but guiding a patient or family to an understanding of death or illness.
> 
> The physician’s duty is not to stave off death or return patients to their old lives, but to take into our arms a patient and family whose lives have disintegrated and work until they can stand back up and face, and make sense of, their own existence.
> 
> -Paul Kalanithi

It’s not a job for him, it’s a calling that commands an extreme level of passion for what he’s doing and a duty towards the people impacted.

I think about the people in my life who seem to operate this way—people who love the work for the work itself and not for the tangible and intangible things their jobs provide them.

I think about my dad who’s still fixing cars every day despite being 70 years old. He could have a bad back or it could be 100 degrees outside and he’ll just get up and go to work without any fuss or spectacle. He just loves fixing cars and there’s nothing he’d rather do despite the physical toll.

When you talk to people like my dad, they emanate a uniquely quiet confidence. They don’t feel any need to announce that they’re happy or successful, they just love what they do and find real purpose in it.

No signaling. No need for external validation. No concern about their perceived place in society.

They've found a calling where they just have a pure, unadulterated love for their craft and the people they help.

It's incredibly refreshing to meet people like this and I think we're all on our own personal journeys to figure out what gives us this sense of purpose and bliss.

As for my personal journey figuring this out, well…I'll save that long-winded topic for another newsletter 🙂

### 2: Our p**lans are incredibly fragile**
_When Breath Becomes Air_ begins with Paul checking himself into Stanford Medical Center after experiencing weeks of severe back pain.

X-Rays couldn't find anything wrong with his back so the doctors decided to scan his chest.

That morning, Paul flips through images of his own CT scan and realizes that his lungs are filled with tumors.

At only 36 years old, it's confirmed that he has Stage IV lung cancer.

Years of being a neurosurgeon had earned him the respect of his peers, prestigious medical awards, and job offers from major universities.

Paul was just reaching the peak of his career. He was looking forward to finishing his grueling residency, spending more time with his wife Lucy, and starting the family they had always dreamed of.

Paul was on the cusp of finally living the life he had planned for himself, but now signs of advanced cancer have mercilessly snatched that beautiful future away from him.

> And with that, the future I had imagined, the one just about to be realized, the culmination of decades of striving, evaporated.
> 
> -Paul Kalanithi

I put myself in Paul’s shoes and tried to imagine what it would be like to have your life crumble in an instant.

How would I respond? How would I tell my friends and family? How would I live my life with the time remaining?

I couldn't arrive at any conclusive answers. These were such heavy and profound questions that I simply wasn't able to work through in the abstract.

The closet I came was thinking about a friend of mine who passed away before his senior year of college.

The weekend of his funeral was a blur but I do remember how all of us had a distinct moment where the gravity of this tragedy fully hit home.

For me, it was when one of his family members showed me a toy car my friend had built when he was a little kid. His family had kept it this entire time with intention to gift it back to him the day he graduated from college.

It was going to be their way of saying: “We’ve always believed in you and we’re so proud of how far you’ve come.”

My friend was only a year away from graduating. Realizing how that beautiful moment would never happen absolutely broke my heart.

He had so much more life left to live, so many things left to accomplish, and so many more memories to create with friends and loved ones.

It made me realize how our lives can be so full promise and yet so incredibly fragile.

> My life had been building potential, potential that would now go unrealized. I had planned to do so much, and I had come so close…The lung cancer was confirmed. My carefully planned and hard-won future no longer existed.
> 
> -Paul Kalanithi

We spend so much time hoping and planning. We prepare for our careers, we look forward to buying our dream home, and we daydream about starting a family with the person we love.

But what do we do if life decides it doesn’t care about our best-laid plans? What do we do when the goals we had worked so hard towards are no longer within reach? How do we still find joy, value, and purpose?

> I would have to learn to live in a different way, seeing death as an imposing itinerant visitor but knowing that even if I'm dying, until I actually die, I am still living.
> 
> -Paul Kalanithi

Paul adopts an elegantly simple approach: Just keep going because you don't really have a choice.

I’ll pause here and preface that I don’t want to diminish what people are going through with yet another platitude about resilience.

I don’t think Paul’s suggesting that we skip the necessary time to properly mourn, grieve, and appreciate the difficulties we encounter.

Rather, I think he’s simply offering that—when we’re ready—there’s tremendous value in continuing to press on…according to plan or not.

> I can’t go on. I’ll go on.
> -Paul Kalanithi (quoting Samuel Beckett)

### 3: Be intentional about the good things in your life

> Flush in the face of mortality, many decisions became compressed, urgent and unreceding. Foremost among them for us: Should Lucy and I have a child?
> -Paul Kalanithi

In the second half of his memoir, Paul and his wife Lucy decide to have a child despite the severity of his diagnosis. They believed that fulfilling their dream of becoming parents would bring them much-needed joy during a difficult time.

I thought about this choice Paul and Lucy made for awhile.

It's true that the joy of bringing life into this world would give them a brief respite from the crushing burden of facing Paul's mortality every single day.

But at what cost?

Paul would likely never see his daughter grow up, Lucy would need to raise her on her own and their daughter would up grow only knowing who her father was through the words in his memoir.

Paul weighed these choices too, agonizing over the emotional toll this would have on his family.

> Lucy hoped I had years left, but understanding my prognosis, she felt that the choice — whether to spend my remaining time as a father — should be mine.
> 
> “What are you most afraid or sad about?” she asked me one night as we were lying in bed.
> 
> “Leaving you,” I told her.
> 
> "Will having a newborn distract from the time we have together?” [Lucy] asked. “Don’t you think saying goodbye to your child will make your death more painful?”
> 
> “Wouldn’t it be great if it did?” I said.
> 
> Lucy and I both felt that life wasn’t about avoiding suffering.
> 
> -Paul Kalanithi

I took a step back from my callous cost-benefit analysis and realized that it's not my place to disparage this deeply personal decision.

Instead, I think there's something beautiful about Paul and Lucy deciding that living life wasn't a perpetual game of making optimizations.

It wasn't about minimizing loss. It wasn't about properly weighing pros and cons.

Really living life means knowing what gives you joy and meaning and then being intentional when moving towards those things.

I think about how that approach translates to my personal life and I land on the realization of how much my friends and family mean to me.

How often do I tell them how grateful I am that they’re in my life? How often do I reach out and just check up on how they're doing? How often am I there with them and actually present instead of thinking ahead about what’s next for me?

Honestly, probably not often enough.

Stumbling into my mid-twenties during a pandemic taught me how the time we have with loved ones is super valuable but also increasingly rare and not guaranteed. Whether it's our life plans changing, people moving farther away, or just more real-world responsibilities piling up, finding time to really connect with the people we love only gets harder over time.

If there’s one thing I want to do this year, it’s to be more intentional about the really great relationships I have today. I want to put in the time and energy to continue evolving the relationships that mean most to me.

One of my fondest memories happened a few years ago when a group of us were just sitting inside a McDonalds playing cards at 3 in the morning. All things considered, it was a pretty uneventful evening and I'd be surprised if anyone else ascribes the same value to this memory.

But I remember having one of those out of body experiences where I soaked in our laughter, our conversation, and connection with one another. I zoomed out and thought to myself: "Wow, I'm so happy to have this moment with these people."

It was an organic moment of just pure joy and gratitude. A moment of just truly being present with people you love.

This was my favorite lesson from _When Breath Becomes Air._

Rather than fixating on a plan to optimize our lives, living life means finding and cherishing these moments that make you truly happy.

# **Conclusion**

Paul's daughter Cady was born on July fourth only a few days after Paul was temporarily released from the hospital. Despite his cancer stabilizing a bit, Paul knew that his condition could deteriorate quickly.

Paul, Lucy, and their daughter Cady

![https://cdn.substack.com/image/fetch/w_1100,c_limit,f_auto,q_auto:good,fl_progressive:steep/https%3A%2F%2Fbucketeer-e05bbc84-baa3-437e-9518-adb32be77984.s3.amazonaws.com%2Fpublic%2Fimages%2F2078c7a9-eef1-4828-975a-85d1ae296aa0_1500x1000.jpeg](https://cdn.substack.com/image/fetch/w_1100,c_limit,f_auto,q_auto:good,fl_progressive:steep/https%3A%2F%2Fbucketeer-e05bbc84-baa3-437e-9518-adb32be77984.s3.amazonaws.com%2Fpublic%2Fimages%2F2078c7a9-eef1-4828-975a-85d1ae296aa0_1500x1000.jpeg)

He returns home grateful for how much joy Cady has brought to his life. He ends his part of the book with a message to her:

> [Y]ou filled a dying man's days with a sated joy, a joy unknown to me in all my prior years, a joy that does not hunger for more and more but rests, satisfied.
> 
> In this time, right now, that is an enormous thing.
> 
> -Paul Kalanithi

Paul died March 9, 2015 only five months after Cady's birth.

His wife Lucy finishes the book with an epilogue describing the circumstances of Paul's death.

Around Christmas, Paul's treatments stop working and the cancer starts spreading to his brain. Knowing he had little time left, Paul becomes myopically focused on finishing his book, hoping it will leave a lasting account for his daughter.

It was incredible to read Lucy's reflection on Paul's writing and I'll quote her directly since she sums up how powerful this book was better than I can.

> Paul faced each stage of his illness with grace—not with bravado or a misguided faith that he would "overcome" or "beat" cancer but with an authenticity that allowed him to grieve the loss of the future he had planned and forge a new one...
> 
> Even while terminally ill, Paul was fully alive; despite physical collapse, he remained vigorous, open, full of hope not for an unlikely cure but for days that were full of purpose or meaning.
> 
> -Lucy Kalanithi

They say you don't really know someone's true character until they go through adversity. With Paul, we witnessed a man who faced his own mortality and the destruction of his future.

But despite what he faced every single day, he continued to display an incredible amount of grace, strength, and integrity.

> What happened to Paul was tragic, but he was not a tragedy
> 
> -Lucy Kalanithi

I hope I can face challenges in my life with even half the poise that Paul did.
